app/services/clients/stripe/StripeDirector.class.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+const Joi = require('joi')
+
+const schema = {
+  first_name: Joi.string().required(),
+  last_name: Joi.string().required(),
+  email: Joi.string().required(),
+  dob_day: Joi.number().integer().strict().min(1).max(31),
+  dob_month: Joi.number().integer().strict().min(1).max(12),
+  dob_year: Joi.number().integer().strict().min(1000).max(9999),
```

Comment:
Shouldn't it be min(1900).max(2020) or something like that?

Reply:
yeah good point! updated to `min(1900).max(2100)`. DOB validation should prevent if year is in future.